src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -256,6 +256,15 @@ export function getFeatureInstallWrapperScript(feature: Feature, featureSet: Fea
 	const documentation = escapeQuotesForShell(feature.documentationURL ?? '');
 	const optionsIndented = escapeQuotesForShell(options.map(x => `    ${x}`).join('\n'));
 
+	let warningHeader = '';
+	if (feature.deprecated) {
+		warningHeader += `(!) WARNING: Using the deprecated Feature "${feature.id}". This Feature will no longer receive any further updates/support.\n`;
+	}
+
+	if (feature?.legacyIds && feature.legacyIds.length > 0 && feature.id !== feature.currentId) {
+		warningHeader += `(!) WARNING: Using the old Feature ID, update the reference to "${feature.currentId}".`;
```

Comment:
```suggestion
		warningHeader += `(!) WARNING: devcontainer.json references a legacy ID. Update the reference to '${feature.currentId}'.`;
```

Reply:
Umm just merged John's suggestion. Let me know what you think of that.

> `(!) WARNING: This feature has been renamed. Please update the reference in devcontainer.json to "${feature.currentId}".`;

Reply:
Oh I missed his comment. That looks good to me.